google/cloud/aiplatform/experimental/vertex_model/serializers/pandas.py

```diff
@@ -29,7 +30,9 @@
     )
 
 
-def _serialize_dataframe(artifact_uri: str, obj: pd.DataFrame, temp_dir: str) -> str:
+def _serialize_dataframe(
```

Comment:
Consider writing  unit test for `_serialize_dataframe`, `_deserialize_dataframe`, and `make_source` or create a ticket and add `# TODO: b/1234` in those location, depending on time constraints.  

Here's an example of mocking `storage.Client`: https://github.com/googleapis/python-aiplatform/blob/master/tests/unit/aiplatform/test_training_jobs.py#L216

Reply:
Yup, working on these today but will create a bug for doing so right now.